larray/inout/utils.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import, print_function
+
+import os
+from collections import OrderedDict
+from larray.core.axis import Axis
+from larray.core.group import Group
+from larray.core.array import LArray
+
+
+def _get_index_col(nb_axes=None, index_col=None, wide=True):
+    if not wide:
+        if nb_axes is not None or index_col is not None:
+            raise ValueError("`nb_axes` or `index_col` argument cannot be used when `wide` argument is False")
+
+    if nb_axes is not None and index_col is not None:
+        raise ValueError("cannot specify both `nb_axes` and `index_col`")
+    elif nb_axes is not None:
+        index_col = list(range(nb_axes - 1))
+    elif isinstance(index_col, int):
+        index_col = [index_col]
+
+    return index_col
+
+
+_allowed_types = (LArray, Axis, Group)
+
+
+class FileHandler(object):
```

Comment:
not really an utility class (which should be usable in a lot of different contexts). This is a base class, so move this to a module named "base", "core", "abstractbases" or something like that.